priv/repo/migrations/20240525065205_update_user_session.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Dbservice.Repo.Migrations.UpdateUserSession do
+  use Ecto.Migration
+
+  def change do
+    alter table(:user_session) do
+      add :user_activity_type, :string
+      remove :session_occurrence_id
+      add :session_occurrence_id, references(:session_occurrence)
+      add :session_id, references(:session)
+      remove :user_id
+      add :user_id, references(:user)
```

Comment:
Same question here

Reply:
Like you mentioned, `user_id` did not have any reference